string camera_topic
---
sensor_msgs/PointCloud2 cloud_out
